apps/user/models/user.go

```diff
@@ -12,6 +14,17 @@ import (
 	"github.com/iReflect/reflect-app/db/models/fields"
 )
 
+//Email ...
+type Email struct {
+	Email string `json:"email"`
+}
+
+//TimeProviderConfig ...
+type TimeProviderConfig struct {
+	Data Email  `json:"data"`
```

Comment:
This should be an interface and we should handle the type of data dynamically based on the type field.
```suggestion
	Data interface{}  `json:"data"`
```

Reply:
changed